.github/hub/sklearn_parser.py

```diff
@@ -0,0 +1,129 @@
+from inspect import getmembers, isfunction, signature, _empty
+from sklearn import metrics
+from cookiecutter.main import cookiecutter
+import subprocess
+
+classification_metrics = ['accuracy_score',
+ 'auc',
+ 'average_precision_score',
+ 'balanced_accuracy_score',
+ 'brier_score_loss',
+ 'classification_report',
+ 'confusion_matrix',
+ 'dcg_score',
+ 'det_curve',
+ 'f1_score',
+ 'fbeta_score',
+ 'hamming_loss',
+ 'hinge_loss',
+ 'jaccard_score',
+ 'log_loss',
+ 'matthews_corrcoef',
+ 'multilabel_confusion_matrix',
+ 'ndcg_score',
+ 'precision_recall_curve',
+ 'precision_recall_fscore_support',
+ 'precision_score',
+ 'recall_score',
+ 'roc_auc_score',
+ 'roc_curve',
+ 'top_k_accuracy_score',
+ 'zero_one_loss']
+
+
+classification_features = {
+    "y_true": ['datasets.Sequence(datasets.Value("int32"))', 'datasets.Value("int32")'], 
+    'y_pred': ['datasets.Sequence(datasets.Value("int32"))', 'datasets.Value("int32")'],
```

Comment:
note that some classification scorers require a soft value rather than an integer, e.g. `roc_auc_score`

Reply:
I had a quick look: those usually don't use `y_pred` but `y_score` (`roc_auc_score`),  `y_proba` (`brier_score_loss`) , `y_decision` (`hinge_loss`) and you can see these are defined as `float`. I double checked a few and it seemed to be consistent with the variable name. Let me know if you think i misclassified one.